### PR TITLE
feat(skymp5-server): add base Actor Values to VisitProperties

### LINKED ISSUE #186: Add base Actor Values to `VisitProperties` for `MpActor`
This would add them to `createActor` message.

Note: We should not be able to see Actor Values of other players in `createActor` message on the client.

Note(2): Use `GetBaseActorValues` to get actual values, do NOT modify changeForm on this step. Always recalculate values by calling `GetBaseActorValues`

Note(3): This task does NOT include adding percentage of attributes to `VisitProperties`

### PR DESCRIPTION
**Pull request by d0gge**
closes #186 
Added `baseActorValues` to `VisitProperties` for `MpActor`.

### CODE REVIEW

**Review by Pospelove — CHANGES_REQUESTED** [review]
1. `GetLook` may return nullptr so I think we can pass 0 as raceIdOverride in this case
2. We should call `visitor` for each actor value. See where `VisitProperties` is used to figure out the meaning.

**Comment by d0gge** [comment]
WIP

**Comment by Milord-Freeman** [comment]
The code looks acceptable, but our code coverage tests are failing. It is necessary to check the testing and also provide the clang-format formatting.

**Review by Milord-Freeman — APPROVED** [review]

**Comment by Pospelove** [comment]
@d0gge @Milord-Freeman This task doesn't include modifying the changeform. Please let me know if I am out of context.

File: skymp5-server/cpp/server_guest_lib/MpChangeForms.cpp [hunk 1]
```diff
@@ -37,7 +37,6 @@ nlohmann::json MpChangeForm::ToJson(const MpChangeForm& changeForm)
   res["healthPercentage"] = changeForm.healthPercentage;
   res["magickaPercentage"] = changeForm.magickaPercentage;
   res["staminaPercentage"] = changeForm.staminaPercentage;
-
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/MpChangeForms.cpp:40]: Please remove unnecessary change

File: skymp5-server/cpp/server_guest_lib/MpChangeForms.h [hunk 2]
```diff
@@ -46,7 +46,6 @@ class MpChangeFormREFR
   float healthPercentage = 1.0f;
   float magickaPercentage = 1.0f;
   float staminaPercentage = 1.0f;
-
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/MpChangeForms.h:49]: And here

File: skymp5-server/cpp/server_guest_lib/MpActor.cpp [hunk 3]
```diff
@@ -49,9 +50,36 @@ void MpActor::SetEquipment(const std::string& jsonString)
...
 {
+  auto baseId = MpObjectReference::GetBaseId();
+  uint32_t raceId = GetLook() ? GetLook()->raceId : 0;
+
+  auto baseActorValues = GetBaseActorValues(baseId, raceId);
+
   MpObjectReference::VisitProperties(visitor, mode);
   if (mode == VisitPropertiesMode::All && IsRaceMenuOpen())
     visitor("isRaceMenuOpen", "true");
+
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/MpActor.cpp:61]: These lines duplicate actor values list from `struct BaseActorValues`
Every time we modify this struct, we will have to also modify `VisitProperties` which we will likely forget to do.
Consider removing this logic duplication between files in any possible way.
My idea: Create `VisitBaseActorValues` method in `struct BaseActorValues` and move these lines into it. That's how we can at least place all code logic duplication into one file. It would be much better.

**Review by Pospelove — CHANGES_REQUESTED** [review]

**Review by Pospelove — CHANGES_REQUESTED** [review]
There is still duplication. We shouldn't list actor values in more than on file.

File: skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h [hunk 4]
```diff
@@ -3,15 +3,46 @@
 
 struct BaseActorValues
 {
+
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h:6]: excess whitespace

File: skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h [hunk 5]
```diff
@@ -3,15 +3,46 @@
...
+  float staminaRate = 100;
   float magickaRate = 100;
   float healRateMult = 100;
   float staminaRateMult = 100;
   float magickaRateMult = 100;
+
+  using PropertiesVisitor =
+    std::function<void(const char* propName, const char* jsonValue)>;
+
+  std::string BaseActorValuesToString(float attribute)
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h:20]: You can use `to_string` directly instead of `BaseActorValuesToString`

File: skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h [hunk 6]
```diff
@@ -3,15 +3,46 @@
 
 struct BaseActorValues
 {
+   
   float health = 100;
   float stamina = 100;
   float magicka = 100;
   float healRate = 100;
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h:10]: 100 is too much for rates. see https://elderscrolls.fandom.com/wiki/Console_Commands_(Skyrim)/ActorValues

**Review by Pospelove — CHANGES_REQUESTED** [review]

File: skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h (hunk 6 above)
Comment (d0gge) [reply skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h]: Done @Pospelove

**Review by Pospelove — CHANGES_REQUESTED** [review]
I forgot one thing:
> Note: We should not be able to see Actor Values of other players in createActor message on the client.

We need to check that `mode` is `VisitPropertiesMode::All`. It would make these variables invisible to other players - as the task says.

**Review by Pospelove — APPROVED** [review]

**Comment by d0gge** [comment]
Done @Pospelove
